react/features/notifications/components/web/Notification.js

```diff
@@ -143,7 +143,8 @@ class Notification extends AbstractNotification<Props> {
                             if (this.props.customActionHandler[customActionIndex]()) {
                                 this._onDismissed();
                             }
-                        }
+                        },
+                        testId: this.props.customActionNameKey[customActionIndex]
```

Comment:
Isn't this.props.customActionNameKey[customActionIndex the same as customAction ?

Reply:
Oh.. of course. You're right. Thank you for your review.